src/app/contact-page/contact-page.component.css

```diff
@@ -0,0 +1,17 @@
+#form-container {
```

Comment:
We should try to keep styles consistent. Use the `.box` class and `.page-content` class for this stuff.